src/codegen/mod.rs

```diff
@@ -630,7 +630,18 @@ impl CodeGenerator for Var {
             attrs.push(attributes::doc(comment));
         }
 
-        let ty = self.ty().to_rust_ty_or_opaque(ctx, &());
+        let var_ty = if let Some(var_ty) = self.ty(ctx) {
+            var_ty
+        } else {
+            // FIXME: Parse/output macro variables as the last step when all types are known.
```

Comment:
I mean, this is not possible, I believe, because the type the macro refers to by definition depends on context at the time of expansion.

Reply:
The time of expansion is after everything is parsed, so it should be the last type with the corresponding name.

Reply:
To be clear, `bindgen` currently is not doing the right thing for the following:

```c
#define MY_INT 1
#define MY_INT 2
```

In C, `MY_INT` expands to `2`. `bindgen` currently generates `const MY_INT: u32 = 1`.